script.js

```diff
@@ -77,3 +77,107 @@ function myFunction(x) {
+  },
+  liveVersion: 'https://samiullah997.github.io/Portfolio-Mobile/',
+  liveSource: 'https://github.com/samiullah997/Portfolio-Mobile',
+};
+const showModal = document.getElementById('show-modal');
+const showModal1 = document.getElementById('show-modal-1');
+const showModal2 = document.getElementById('show-modal-2');
+const showModal3 = document.getElementById('show-modal-3');
+const showModal4 = document.getElementById('show-modal-4');
+const showModal5 = document.getElementById('show-modal-5');
+const showModal6 = document.getElementById('show-modal-6');
+const closeModal = document.getElementById('close-modal');
+showModal.addEventListener('click', () => {
+  document.getElementById('modal').classList.add('active');
+  document.getElementById('postTitle').innerText = showItemInfo.title;
+  document.getElementById('tech1').innerHTML = showItemInfo.technologies.tech1;
+  document.getElementById('tech2').innerHTML = showItemInfo.technologies.tech2;
+  document.getElementById('tech3').innerHTML = showItemInfo.technologies.tech3;
+  document.getElementById('description').innerText = showItemInfo.description;
+  document.getElementById('image').style.backgroundImage = `url('${showItemInfo.image}')`;
+  document.getElementById('liveButton').setAttribute('href', showItemInfo.liveVersion);
+  document.getElementById('liveSource').setAttribute('href', showItemInfo.liveSource);
+});
+showModal1.addEventListener('click', () => {
+  document.getElementById('modal').classList.add('active');
+  document.getElementById('postTitle').innerText = showItemInfo.title;
+  document.getElementById('tech1').innerHTML = showItemInfo.technologies.tech1;
+  document.getElementById('tech2').innerHTML = showItemInfo.technologies.tech2;
+  document.getElementById('tech3').innerHTML = showItemInfo.technologies.tech3;
+  document.getElementById('description').innerText = showItemInfo.description;
+  document.getElementById('image').style.backgroundImage = `url('${showItemInfo.image}')`;
+  document.getElementById('liveButton').setAttribute('href', showItemInfo.liveVersion);
+  document.getElementById('liveSource').setAttribute('href', showItemInfo.liveSource);
+});
+showModal2.addEventListener('click', () => {
+  document.getElementById('modal').classList.add('active');
+  document.getElementById('postTitle').innerText = showItemInfo.title;
+  document.getElementById('tech1').innerHTML = showItemInfo.technologies.tech1;
+  document.getElementById('tech2').innerHTML = showItemInfo.technologies.tech2;
+  document.getElementById('tech3').innerHTML = showItemInfo.technologies.tech3;
+  document.getElementById('description').innerText = showItemInfo.description;
+  document.getElementById('image').style.backgroundImage = `url('${showItemInfo.image}')`;
+  document.getElementById('liveButton').setAttribute('href', showItemInfo.liveVersion);
+  document.getElementById('liveSource').setAttribute('href', showItemInfo.liveSource);
+});
```

Comment:
- [ ] Please make sure that the pop-up window on both mobile and desktop version are displayed correctly in the right place so please try to fix that 
![image](https://user-images.githubusercontent.com/61790587/179959767-043c4a73-8abb-4e52-a90f-99c5075c192e.png)
![image](https://user-images.githubusercontent.com/61790587/179959835-82c26aec-19c3-4fa2-8b2a-ee3b95e21027.png)